tgui/packages/tgui/interfaces/Hypertorus.js

```diff
@@ -0,0 +1,306 @@
+import { sortBy, map, filter } from 'common/collections';
+import { flow } from 'common/fp';
+import { toFixed } from 'common/math';
+import { useBackend } from '../backend';
+import { AnimatedNumber, Button, Flex, Input, LabeledList, ProgressBar, Section, Table, NumberInput } from '../components';
+import { getGasColor, getGasLabel } from '../constants';
+import { Window } from '../layouts';
+import { formatSiUnit, formatSiBaseTenUnit } from '../format';
+
+export const Hypertorus = (props, context) => {
+  const { act, data } = useBackend(context);
+  const filterTypes = data.filter_types || [];
+  const {
+    energy_level,
+    core_temperature,
+    internal_power,
+    power_output,
+    heat_limiter_modifier,
+    heat_output,
+    heat_output_bool,
+    heating_conductor,
+    magnetic_constrictor,
+    fuel_injection_rate,
+    moderator_injection_rate,
+    current_damper,
+    power_level,
+    iron_content,
+    integrity,
+    start_power,
+    start_cooling,
+    start_fuel,
+    internal_fusion_temperature,
+    moderator_internal_temperature,
+    internal_output_temperature,
+    internal_coolant_temperature,
+    waste_remove,
+  } = data;
+  const fusion_gases = flow([
+    filter(gas => gas.amount >= 0.01),
+    sortBy(gas => -gas.amount),
+  ])(data.fusion_gases || []);
+  const moderator_gases = flow([
+    filter(gas => gas.amount >= 0.01),
+    sortBy(gas => -gas.amount),
+  ])(data.moderator_gases || []);
+  const fusionMax = Math.max(1, ...fusion_gases.map(gas => gas.amount));
+  const moderatorMax = Math.max(1, ...moderator_gases.map(gas => gas.amount));
+  return (
+    <Window
+      width={500}
+      height={600}
+      scrollable
+      resizable>
+      <Window.Content>
+        <Section title="Fusion Reactor">
```

Comment:
Remove this unnecessary wrapping section. Perhaps what you need is `<Window title="Fusion Reactor">`.